Main/Include/miscitem.h

```diff
@@ -203,6 +203,7 @@ ITEM(scrollofearthquake, scroll)
 {
  public:
   virtual void FinishReading(character*);
+  static void EarthQuakeMagic();
```

Comment:
Great! I never made myself unify this duplicate code. :)

BTW, xinrochghost also uses (another duplicate of) earthquake when killed.

Reply:
done! 
The only difference is when xinrochghost will say his final words (after the earthquake completes), but I kept the details on the messages about hitting NPCs with ceiling bricks :>